src/frontend/src/pages/ChangeRequestsPage/ChangeRequestsTable.tsx

```diff
@@ -120,12 +121,15 @@ const ChangeRequestsTable: React.FC = () => {
         title={'Change Requests'}
         previousPages={[]}
         actionButton={
-          <ActionButton
-            link={routes.CHANGE_REQUESTS_NEW}
-            icon={faPlus}
-            text={'New Change Request'}
-            disabled={auth.user?.role === 'GUEST'}
-          />
+          <Link
+            className={'row py-auto px-3 '}
+            to={routes.CHANGE_REQUESTS_NEW}
+            style={{ textDecoration: 'none', pointerEvents: false ? 'none' : undefined }}
+          >
+            <Button disabled={auth.user?.role === 'GUEST'} endIcon={<Add />} variant="text">
+              New Change Request
+            </Button>
+          </Link>
```

Comment:
```
`        <NERButton
          component={Link}
          to={routes.CHANGE_REQUESTS_NEW}
          variant="contained"
          disabled={auth.user?.role === 'GUEST'}
          startIcon={<Add />}
        >`
```
Do it this way instead. You might need to merge the latest version of [#142-MUI-migration](https://github.com/Northeastern-Electric-Racing/FinishLine/tree/%23142-MUI-migration)  into your branch to get access to the NERButton Component. I made it for the MUI transition, so the button styling would be standard across the whole app.

Reply:
Also, just in general, for all these MUI transition tickets we want the new stuff to look as close to the old stuff as possible, so for instance here, just make sure the new button looks as close to the old one as possible (in size, color, icons, etc.).